components/Common/CodeTabs/index.tsx

```diff
@@ -23,6 +23,7 @@ const CodeTabs: FC<PropsWithChildren<CodeTabsProps>> = ({
   <Tabs
     {...props}
     className={styles.root}
+    forceDarkTheme={true}
```

Comment:
Why? Just apply the styles by default. Our Codebox is always dark

Reply:
because the tabs component could be used in a location other than the codeBox

Reply:
This fix should not be in the Tabs component. But directly within the CodeTabs component styles. Please remove these changes from the Tabs component

Reply:
Ok, I see what you mean. I allow with you it's will more clear.